QuoteEngine/README.md

```diff
@@ -1,30 +1,177 @@
+   <br>
+      This will create a user named "botnick". We will use this user,
+      <br>
+      to connect to mysql and make changes in database named "quotesdb".
+      <br>
+      For that, we need to set rights.
+      
+   ```mysql> GRANT ALL PRIVILEGES ON quotesdb.* TO '<botnick>'@'localhost';```
+   <br>
+      This will grant all right on db "quotesdb" to "botnick" user, connecting from localhost.
+    
+    
+   Now we are ready to move on to the next step. :)
+   
+   
+2. Now, you have to create the tables into the database which you made.
+   <br>
+   For this, you can download the pre-made script - you can find it in the sql folder, named "quotes.sql".
+   Download it (example: wget) and run the following command with your shell user:
+   <br>
+      ```$ mysql -u <user> -p quotesdb < quotes.sql```
+      <br>
+         The <user> is what you have created at point 1. :)
+
+
+   At this point you are ready with the preparations and you can move to the next chapter. :)
+
+
+Setting up the tcl script, edit the config(s)
+---------------------------------------------
+
+1. Download the script (QuoteEngine.tcl) and put that into the eggdrop's  scripts directory.
+2. Download the settings file (QuoteEngine-settings.sample.tcl) rename it to "QuoteEngine-settings.tcl" and edit it!
+   <br>
+   The settings file needs to be edited. If you followed the guide, you won't have any problems to fill it out. :)
+3. Open your bot's configuraton file and put the following like to the end of the file:
+     <br>
+      ```source "scripts/QuoteEngine.tcl"```
+      <br>
+4. Telnet to your bot (or use dcc chat), or however you go to your bot's console and rehash your bot.
+   You need to see this line in the console:
+   <br>
+      `"QuoteEngine 1.3 loaded"`
+    <br>
+   In case you cannot see it, you did something wrong. Check again the guide. :) 
+
+
+Usage
+---------------------------------------------
+
+1. You have to set "+quoteengine" flag to the channel where you want to enable to use the commands provided by the script.
+   <br>
+   You can do this via the bot's console with command:
+      ```.chanset <#channelname_here> +quoteengine```
+      <br>
+2. If you did everything right, now you will be able to use commands as:
+
+```
+!addquote <text>        -- add a quote
+!getquote <#number>     -- get the quote of #
+!randquote              -- get a random quote
+!delquote <#number>     -- delete a quote
+!quotehelp              -- list of all available command
+```
+
+
+Note: some of the commands are limited bot owners, masters, etc..
+<br>
+Check QuoteEngine.tcl for further details.
+   (
+    You need to check the "bind pub" lines. :)
+    m,f,o,v are flags (rights) which you can have in the bot
+   )
+
 
 Webpage setup
--------------
+---------------------------------------------
+
+!! The php part - so the webiste - won't work, because the code is for php5, so on php7 it wont work.
```

Comment:
:D

Don't think I've really touched PHP since 5, myself

Reply:
it's sad though. i use the website on my old server (quoteengine is there). i really loves it.
the 3rd party page is not so clear for me though. that twig stuff.. meh.